### PR TITLE
Add PostgreSQL version 10

### PR DESCRIPTION
**Pull request by norshtein**
This PR enables PostgreSQL v10. Three services are added:
- azure-postgresql-10
- azure-postgresql-10-dbms
- azure-postgresql-10-database

They are nearly the same as `azure-postgresql-9-6-*`, except that `version` in `ServiceProperties.Extended` is changed.

### CODE REVIEW

**Comment by krancour** [comment]
I'm stopping short of LGTM'ing, but this is mostly ok with me.

One thing though...

It was deliberate that we included versions in service names. It allowed us to peg the lifecycle of services in the catalog to that of managed services in Azure. Eventually (distant future) Azure won't offer PostgreSQL 9.6 anymore and the corresponding service will have to be EOL'ed as well. i.e. We planned on services being retired and replaced over time. That creates natural opportunities to introduce breaking changes (to services; not to OSBA as a whole) if they are desired or needed.

So all I'm pointing out here is that _if_ there are things that you'd like to do with the PostgreSQL 10 services that would be breaking deviations from how the PostgreSQL 9.6 services work, consider copying and modifying the service managers under a new package. (To be clear, I'm suggesting this course of action only if you have identified a specific need for it.)

Also, I think you should let these new services be "preview" for the next release. I say this because I do think that, short term, you might want the flexibility to make changes as described above.

**Comment by norshtein** [comment]
Hi @krancour , thanks for the suggestion! I'm going to make PostgreSQL 10 services to be "preview" and refactor the code later if we find anything in PostgreSQL 10 will break PostgreSQL 9.6.

**Comment by krancour** [comment]
@norshtein sgtm.

**Comment by krancour** [comment]
@norshtein lgtm after you change this to preview.

**Comment by norshtein** [comment]
Hi @krancour , I have changed postgreSQL v10 to preview, could you please have a review on it?

File: docs/modules/postgresql.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+*Note: PostgreSQL version 9.6 is STABLE, and PostgreSQL version 10 is in PREVIEW*
+
 Open Service Broker for Azure contains three Azure Database for PostgreSQL services. These services enable you to select the most appropriate provision scenario for your needs. These services are:
 
 | Service Name | Description |
 |--------------|-------------|
-| `azure-postgresql-9-6` | Provision both an Azure Database for PostgreSQL Database Management System (DBMS) and a database. |
-| `azure-postgresql-9-6-dbms` | Provision only an Azure Database for PostgreSQL DBMS. This can be used to provision multiple databases at a later time. |
-| `azure-postgresql-9-6-database` | Provision a new database only upon a previously provisioned DBMS. |
+| `azure-postgresql-9-6` and `azure-postgresql-10` | Provision both an Azure Database for PostgreSQL Database Management System (DBMS) and a database. |
```
[the remark this answers is not in the document]
Comment (norshtein) [reply docs/modules/postgresql.md]: Sounds good. I'll polish the doc.

**Comment by norshtein** [comment]
Hi @zhongyi-zhang , are you OK with the change of the doc?

**Pull request merged**